frontend/src/components/templates/Enrollment.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FunctionComponent } from 'react';
+import Form from '../../components/templates/Form';
+import OrganisationSection from '../../components/organisms/form-sections/OrganisationSection';
+import DemarcheSection from '../../components/organisms/form-sections/DemarcheSection';
+import DescriptionSection from '../../components/organisms/form-sections/DescriptionSection';
+import DonneesSection from '../../components/organisms/form-sections/DonneesSection';
+import CadreJuridiqueSection from '../../components/organisms/form-sections/CadreJuridiqueSection';
+import CguSection from '../../components/organisms/form-sections/CguSection';
+import ÉquipeSection from '../../components/organisms/form-sections/ÉquipeSection';
+import { getDefaultDocumentationUrl } from '../organisms/Nav';
+import { FullDataProviderConfiguration } from '../../config/data-provider-configurations';
+import { Remark } from 'react-remark';
+import Link from '../atoms/hyperTexts/Link';
+
+type Props = {
+  target_api: string;
+  configuration: FullDataProviderConfiguration;
+};
+
+const Enrollment: FunctionComponent<Props> = ({
+  target_api,
+  configuration: {
+    email,
+    scopes,
+    groups,
+    editeurs,
+    demarches,
+    cguLink,
+    donneesDescription,
+    cadreJuridiqueDescription,
+  },
+}) => {
+  const rehypeReactOptions = {
+    components: {
+      a: (props: any) => <Link inline {...props} />,
+    },
+  };
+  const DonneesDescription = () => (
+    <Remark rehypeReactOptions={rehypeReactOptions}>
+      {donneesDescription}
+    </Remark>
+  );
+  const CadreJuridiqueDescription = () => (
+    <Remark rehypeReactOptions={rehypeReactOptions}>
+      {cadreJuridiqueDescription}
+    </Remark>
+  );
+
+  return (
+    <Form
+      target_api={target_api}
+      /* @ts-ignore */
```

Comment:
If you're not switching components to TypeScript, I would recommend to make a ticket just after listing all of the `@ts-ignore` components and I'll do the switch myself (as it is one of my main onboarding tasks)